Makefile

```diff
@@ -147,6 +147,26 @@ check-go-version:
 integration-test: integration-test-prereqs
 	./scripts/run-integration-tests.sh
 
+.PHONY: integration-test-suite-1
+integration-test-suite-1: integration-test-prereqs
+	./scripts/run-integration-tests.sh configtx gossip raft
```

Comment:
Instead of hardcoding five suites in the Makefile, how about passing the integration test directories to a single Make target?
e.g. https://chriswiegman.com/2021/08/using-parameters-in-a-make-target/